addons/auth/.eslintrc.js

```diff
@@ -14,7 +14,11 @@ module.exports = {
     // end of babel-config
   },
   plugins: ['ember'],
-  extends: ['eslint:recommended', 'plugin:ember/recommended', 'prettier'],
+  extends: [
+    'eslint:recommended',
+    'plugin:ember/recommended',
+    'plugin:prettier/recommended',
```

Comment:
Do we want prettier rules in `eslint`? `plugin:prettier/recommended` adds all the prettier rules for eslint. I prefer turning off styling rules from `eslint` and just let prettier handle it rather than having prettier rules run by `eslint`.

Reply:
I do agree with @ZedLi opinion here.